File: old/yatian/src/edu/virginia/engine/display/TweenParam.java
package edu.virginia.engine.display;

public class TweenParam {
	private TweenableParams paramToTween;
	private double startVal;
	private double endVal;
	private double time;
	private double percent;
	
	public TweenParam(TweenableParams paramToTween, double startVal, double endVal, double time){
		this.paramToTween=paramToTween;
		this.startVal=startVal;
		this.endVal=endVal;
		this.time=time;
		this.percent=0;
	}
	public TweenableParams getParam(){
		return paramToTween;
	}
	public double getStartVal(){
		return startVal;
	}
	public double getEndVal(){
		return endVal;
	}
	public double getTweenTime(){
		return time;
	}
	public double getPercent(){
		return percent;
	}
}

File: old/oscar/src/edu/virginia/engine/util/Tweening/TweenParam.js
"use strict";

class TweenParam{
  /* Create empty tweens ArrayList */
  constructor(paramToTween, startVal, endVal, endTime){
    this.paramToTween = paramToTween;
    this.startVal = startVal;
    this.endVal = endVal;
    this.endTime = endTime;
  }
  getParam(){return this.paramToTween;}
  getStartVal(){return this.startVal;}
  getEndVal(){return this.endVal;}
  getEndTime(){return this.endTime;}
}
